Makefile

```diff
@@ -1,4 +1,4 @@
-all: vet staticcheck test
+all: vet test
 
 test:
 	GODEBUG=x509sha1=1 go test -covermode=count -coverprofile=coverage.out .
```

Comment:
SHA1 is no longer supported from Go 1.24, we need to refactor to only include SHA1 for supported Go versions and consider removing it entirely in the future.

Currently the Go 1.24 test is failing (you should always test before asking a code review) as result.